.github/workflows/ci-cd-recruitment-service.yml

```
@@ -32,6 +33,11 @@ jobs:
         java-version: '21'
         distribution: 'temurin'
 
+    - name: Set up application.properties
+      run: |
+        mkdir -p "${{ env.SERVICE_NAME }}"/src/main/resources
+        echo "${{ env.APPLICATION }}" > "${{ env.SERVICE_NAME }}/src/main/resources/application.properties"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)